### PR TITLE
fix(ipfo): fix ipfo US order

### PR DESCRIPTION
**Pull request by frenautvh**
See : SUUSAECOMM-2192

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build#5412.0 ✘  
* Stage 1
  * deps ✔ 
* Stage 2
  * build ✘  
  * test ✘

File: client/app/cloud/project/compute/infrastructure/ip/failover/buy/cloud-project-compute-infrastructure-ip-failover-buy-agora.controller.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+            if (result.status !== 200) {
+                return $q.reject(result);
+            }
+            return _.filter(_.get(result, "data.plans"), (offer) => {
+                return /failover/.test(offer.planCode);
+            });
+        });
+    }
+
+    self.getPrice  = (product) => {
```
Comment (antleblanc) [inline client/app/cloud/project/compute/infrastructure/ip/failover/buy/cloud-project-compute-infrastructure-ip-failover-buy-agora.controller.js]: `self.getPrice  =` => `self.getPrice =` (extra space) :)

File: client/app/cloud/project/compute/infrastructure/ip/failover/buy/cloud-project-compute-infrastructure-ip-failover-buy-agora.html [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <div class="oui-modal__footer">
+        <button type="reset"
+                class="oui-button oui-button_secondary"
+                data-ng-click="$ctrl.cancel()">
+            <span data-translate="common_cancel"></span>
```
Comment (antleblanc) [inline client/app/cloud/project/compute/infrastructure/ip/failover/buy/cloud-project-compute-infrastructure-ip-failover-buy-agora.html]: You can apply `data-translate` directive directly on the `<button>` element.

File: client/app/cloud/project/compute/infrastructure/ip/failover/buy/cloud-project-compute-infrastructure-ip-failover-buy-agora.html [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        </button>
+        <button type="submit"
+                class="oui-button oui-button_primary"
+                data-ng-disabled="!$ctrl.product || !$ctrl.quantity || !$ctrl.country">
+            <span data-translate="cpciif_buy_button_generate_bc"></span>
```
Comment (antleblanc) [inline client/app/cloud/project/compute/infrastructure/ip/failover/buy/cloud-project-compute-infrastructure-ip-failover-buy-agora.html]: You can apply `data-translate` directive directly on the `<button>` element.

**Comment by ovh-cds** [comment]
CDS Report build#5415.0 ✘  
* Stage 1
  * deps ✔ 
* Stage 2
  * build ✘  
  * test ✔

**Pull request merged**